addrmgr/addrmanager_test.go

```diff
@@ -105,10 +107,35 @@ func lookupFunc(host string) ([]net.IP, error) {
 	return nil, errors.New("not implemented")
 }
 
+func newAddrManager(t *testing.T, testName string, localSubnetworkID *subnetworkid.SubnetworkID) (*AddrManager, func()) {
```

Comment:
Please give names to the return values, so that it's clear that the returned function is the teardown func